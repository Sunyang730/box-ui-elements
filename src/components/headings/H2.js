// @flow
import * as React from 'react';
import classNames from 'classnames';

import './H2.scss';

type Props = {
    children: React.Node,
    className?: string,
};

const H2 = ({ children, className }: Props) => <h2 className={classNames('bdl-H2', className)}>{children}</h2>;

export default H2;
