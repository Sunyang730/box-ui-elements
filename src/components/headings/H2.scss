@import '../../styles/variables';

h2.bdl-H2 {
    @include font-family;

    color: $bdl-gray;
    font-size: 18px;
    font-weight: bold;
    letter-spacing: .3px;
    line-height: 24px;
}
