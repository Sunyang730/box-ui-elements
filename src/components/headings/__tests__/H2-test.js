// @flow
import * as React from 'react';

import H2 from '../H2';

describe('components/headings/H2', () => {
    const getWrapper = (props = {}) => shallow(<H2 {...props}>foo</H2>);

    test('should render H2', () => {
        const wrapper = getWrapper({ className: 'moo' });

        expect(wrapper).toMatchSnapshot();
    });
});
