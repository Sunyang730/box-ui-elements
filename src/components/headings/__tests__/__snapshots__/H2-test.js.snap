// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/headings/H2 should render H2 1`] = `
<h2
  className="bdl-H2 moo"
>
  foo
</h2>
`;
